content/components/underline-nav.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Underline nav
+description: The UnderlineNav is used to display navigation.
+componentId: underline_nav
+reactId: underline_nav
+figma: https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=18843-67449&t=XqqSolC6AzRO9nae-11
```

Comment:
we could add a `figmaId:  underline_nav` instead of the figma URL, so merging this with the figma automation docs will avoid any possible conflict